### ISSUE TITLE
Bug for next weeks bonus in AH scraper

### ISSUE THREAD

**Issue by Tweska** [m-rtijn/argostime#17]
There is a bug in the AH scraper.

When a product is marked as being on discount next week, it is not parsed correctly.

Examples:
- https://argostime.mrtijn.nl/product/wi510587
- https://argostime.mrtijn.nl/product/wi93224
- https://argostime.mrtijn.nl/product/wi4177

**Comment by m-rtijn** [comment]
Yeah this is a known issue. When products will be discounted in the following week, there is no guarantee that there will be a currently valid price listed.

An example here is the broccoli next week:
![2022-02-27-174410_1252x558_scrot](https://user-images.githubusercontent.com/5114106/155891371-d9bf4ed6-5bd7-4c38-b8bf-d12c922b6a2a.png)
Although there is a normal per kg price listed, the price that normally is scraped is the per 500g price. This also happened with other products when I was initially writing the scraper, for example crates of beer. This week I can't find a lot examples, so maybe they've fixed it, but I'm not sure.
